<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DashBoard</class>
 <widget class="QMainWindow" name="DashBoard">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1193</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Independent National Electoral Commission - OpenVR</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="devPrinter">
    <property name="geometry">
     <rect>
      <x>647</x>
      <y>400</y>
      <width>361</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid grey; background-color:rgb(255, 196, 196);padding:10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="devFP">
    <property name="geometry">
     <rect>
      <x>647</x>
      <y>211</y>
      <width>361</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid grey; background-color:rgb(255, 196, 196);padding:10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="devWebcam">
    <property name="geometry">
     <rect>
      <x>648</x>
      <y>306</y>
      <width>361</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid grey; background-color:rgb(255, 196, 196);padding:10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>416</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Printer</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>761</x>
      <y>225</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Fingerprint Scanner</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>322</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Webcam</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>121</width>
      <height>231</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>resources/logo.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="devRemovable">
    <property name="geometry">
     <rect>
      <x>648</x>
      <y>118</y>
      <width>361</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid grey; background-color:rgb(255, 196, 196);padding:10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>761</x>
      <y>139</y>
      <width>210</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Backup Storage</string>
    </property>
   </widget>
   <widget class="QLabel" name="leFP">
    <property name="geometry">
     <rect>
      <x>761</x>
      <y>254</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect FingerPrint</string>
    </property>
   </widget>
   <widget class="QLabel" name="lePrinter">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>440</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect Printer</string>
    </property>
   </widget>
   <widget class="QLabel" name="leRemovable">
    <property name="geometry">
     <rect>
      <x>761</x>
      <y>165</y>
      <width>210</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect Backup</string>
    </property>
   </widget>
   <widget class="QLabel" name="leWebcam">
    <property name="geometry">
     <rect>
      <x>762</x>
      <y>350</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect Webcam</string>
    </property>
   </widget>
   <widget class="QLabel" name="devPrinter_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>400</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>resources/hp-printer.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="devFP_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>207</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>resources/fp-scanner.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="devRemovable_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>125</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>resources/usb-flash-drive.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="devWebcam_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>310</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>resources/webcam.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>400</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>OpenVR</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>431</width>
      <height>471</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Main Menu</string>
     </attribute>
     <widget class="QCommandLinkButton" name="btnShutdown">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>91</x>
        <y>340</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sans Serif</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&amp;Shutdown</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>resources/icons/lock.png</normaloff>resources/icons/lock.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="btnReports">
      <property name="geometry">
       <rect>
        <x>92</x>
        <y>190</y>
        <width>231</width>
        <height>39</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sans Serif</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>R&amp;eports</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>resources/icons/chart.png</normaloff>resources/icons/chart.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>33</x>
        <y>101</y>
        <width>36</width>
        <height>18</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Courier 10 Pitch</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>[F3]</string>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="btnVotersCard">
      <property name="geometry">
       <rect>
        <x>92</x>
        <y>93</y>
        <width>231</width>
        <height>39</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sans Serif</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&amp;Print Voter's Card</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>resources/icons/voter-card.png</normaloff>resources/icons/voter-card.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>33</x>
        <y>206</y>
        <width>36</width>
        <height>18</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Courier 10 Pitch</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>[F5]</string>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="btnQueueTickets">
      <property name="geometry">
       <rect>
        <x>92</x>
        <y>144</y>
        <width>231</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sans Serif</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&amp;Queue Tickets </string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>resources/icons/ticket-icon.png</normaloff>resources/icons/ticket-icon.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="btnRegistration">
      <property name="geometry">
       <rect>
        <x>79</x>
        <y>30</y>
        <width>241</width>
        <height>51</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active/>
        <inactive/>
        <disabled/>
       </palette>
      </property>
      <property name="font">
       <font>
        <family>Sans Serif</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&amp;Registration</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>resources/icons/camera.png</normaloff>resources/icons/camera.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>36</width>
        <height>48</height>
       </size>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>340</y>
        <width>51</width>
        <height>38</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Courier 10 Pitch</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>[F12]</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>33</x>
        <y>50</y>
        <width>36</width>
        <height>18</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Courier 10 Pitch</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>[F2]</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>33</x>
        <y>156</y>
        <width>36</width>
        <height>18</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Courier 10 Pitch</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>[F4]</string>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="btnBackup">
      <property name="geometry">
       <rect>
        <x>92</x>
        <y>240</y>
        <width>231</width>
        <height>39</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sans Serif</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>DDC &amp;Backup</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>resources/icons/Arrow Down.png</normaloff>resources/icons/Arrow Down.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>33</x>
        <y>251</y>
        <width>36</width>
        <height>18</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Courier 10 Pitch</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>[F6]</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>35</x>
        <y>299</y>
        <width>36</width>
        <height>18</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Courier 10 Pitch</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>[F7]</string>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="btnControlPanel">
      <property name="geometry">
       <rect>
        <x>91</x>
        <y>288</y>
        <width>231</width>
        <height>39</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sans Serif</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&amp;Control Panel</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>resources/icons/control-panel.png</normaloff>resources/icons/control-panel.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="btnWipeOut">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>390</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sans Serif</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:red;</string>
      </property>
      <property name="text">
       <string>Wip&amp;e Data</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>resources/icons/warning.png</normaloff>resources/icons/warning.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andika Basic</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Version 1.9c
 Patched</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QProgressBar" name="batteryBar">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>72</y>
      <width>361</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>52</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Laptop Battery</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbBatteryStatus">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>50</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>charging</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:red;</string>
    </property>
    <property name="text">
     <string>BACKUP OFTEN =&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1193</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAdmin">
    <property name="title">
     <string>&amp;Operator Admin</string>
    </property>
    <addaction name="actionOperator_Registration"/>
    <addaction name="actionChange_Password"/>
    <addaction name="separator"/>
    <addaction name="actionPrinter_Setup"/>
    <addaction name="actionManage_Printers"/>
    <addaction name="separator"/>
    <addaction name="actionRepair_NTFS_Backup_Drive"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionManual"/>
    <addaction name="actionAbout_OpenVR"/>
   </widget>
   <widget class="QMenu" name="menuControl_Panel">
    <property name="title">
     <string>&amp;Control Panel</string>
    </property>
    <addaction name="actionAdministrator_Registration"/>
    <addaction name="actionChange_Admin_Password"/>
    <addaction name="separator"/>
    <addaction name="actionImport_Latest_Backups"/>
    <addaction name="actionImport_Backups"/>
    <addaction name="actionLaunch_Aggregate_AFIS"/>
    <addaction name="actionPrint_Aggregation_Report"/>
    <addaction name="actionBackup_to_CD_DVD"/>
   </widget>
   <addaction name="menuAdmin"/>
   <addaction name="menuControl_Panel"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionManual">
   <property name="text">
    <string>Manual</string>
   </property>
  </action>
  <action name="actionAbout_OpenVR">
   <property name="text">
    <string>About OpenVR</string>
   </property>
  </action>
  <action name="actionPrinter_Setup">
   <property name="text">
    <string>Add New Printer</string>
   </property>
  </action>
  <action name="actionImport_Backups">
   <property name="text">
    <string>Import Backups</string>
   </property>
  </action>
  <action name="actionManage_Printers">
   <property name="text">
    <string>Manage Printers</string>
   </property>
  </action>
  <action name="actionChange_Password">
   <property name="text">
    <string>Change Password</string>
   </property>
  </action>
  <action name="actionOperator_Registration">
   <property name="text">
    <string>Operator Registration</string>
   </property>
  </action>
  <action name="actionAdministrator_Registration">
   <property name="text">
    <string>Administrator Registration</string>
   </property>
  </action>
  <action name="actionChange_Admin_Password">
   <property name="text">
    <string>Change Admin Password</string>
   </property>
  </action>
  <action name="actionLaunch_Aggregate_AFIS">
   <property name="text">
    <string>Launch Aggregate AFIS</string>
   </property>
  </action>
  <action name="actionPrint_Aggregation_Report">
   <property name="text">
    <string>Print Aggregation Report</string>
   </property>
  </action>
  <action name="actionBackup_to_CD_DVD">
   <property name="text">
    <string>Instant Backup</string>
   </property>
  </action>
  <action name="actionImport_Latest_Backups">
   <property name="text">
    <string>Import Latest Backups</string>
   </property>
  </action>
  <action name="actionRepair_NTFS_Backup_Drive">
   <property name="text">
    <string>Repair NTFS Backup Drive</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
